.github/workflows/trivy.yml

```diff
@@ -26,7 +26,7 @@ jobs:
           java-version: '17'
 
       # set up SBT cache
-      - uses: actions/cache@v2
+      - uses: actions/cache@v4
```

Comment:
I've never been 100% sure builds actually read from this cache, but I've never quite gotten around to timing a with/without pair.